packages/reactivity/src/ref.ts

```diff
@@ -67,15 +67,40 @@ export function triggerRefValue(ref: RefBase<any>, newVal?: any) {
   }
 }
 
+/**
+ * Checks if a value is a ref object.
+ *
+ * @param r The value to inspect.
+ */
 export function isRef<T>(r: Ref<T> | unknown): r is Ref<T>
 export function isRef(r: any): r is Ref {
   return !!(r && r.__v_isRef === true)
 }
 
+/**
+ * Creates a mutable reactive ref object from the given "inner value".
```

Comment:
I think it's worth adding that if the value is already a ref, it returns the value directly